maintainers/maintainer-list.nix

```diff
@@ -2745,6 +2745,11 @@
     github = "kiloreux";
     name = "Kiloreux Emperex";
   };
+  kim0 = {
```

Comment:
Do you mind splitting this into a separate commit?

Reply:
Done, thanks @mmahut